app/frontend/src/features/profile/ProfileForm.tsx

```diff
@@ -88,124 +90,126 @@ export default function EditProfileForm() {
               />
             )}
           />
-          <ProfileTextInput
-            label="Gender"
-            name="gender"
-            defaultValue={user.gender}
-            inputRef={register}
-          />
-          <ProfileTextInput
-            label="Occupation"
-            name="occupation"
-            defaultValue={user.occupation}
-            inputRef={register}
-          />
-          <Controller
-            control={control}
-            defaultValue={user.languagesList}
-            name="languages"
-            render={({ onChange }) => (
-              <Autocomplete
-                defaultValue={user.languagesList}
-                disableClearable={false}
-                freeSolo
-                multiple
-                onChange={(_, value) => onChange(value)}
-                open={false}
-                options={[""]}
-                renderInput={(params) => (
-                  <TextField
-                    {...params}
-                    helperText="Press 'Enter' to add"
-                    label="Languages"
-                  />
-                )}
-              />
-            )}
-          />
+          <form onSubmit={onSubmit}>
```

Comment:
How come we need to split this into two forms?

Reply:
It seemed to be the only way I could get "enter" to not submit the whole form (the map is excluded from the form)

Reply:
That means the user can now submit with only their username if they press enter, not sure if we want to allow that 🤔

Reply:
Wdym? It seems to work for me. If I change some fields including name and click enter while typing there, all fields are correctly submitted. 

I couldn't find another way - tried seeing if I could detect the field from `handleSubmit` but it only gives the submit button as the source.

Reply:
What I meant was when you type a new user name then press enter with it focused, it would submit the form. I thought that would only submit the username field on its own but it looks like `useForm` seems to deal with that quite well and correctly forwards all the existing data on the page on too.